src/test/java/com/puppycrawl/tools/checkstyle/grammars/AstRegressionTest.java

```diff
@@ -218,7 +218,7 @@ private static void verifyAstRaw(String expectedTextPrintFileName, String actual
                 actualContents);
     }
 
-    private static final class AssertGeneratedJavaLexer extends GeneratedJavaLexer {
+    public static final class AssertGeneratedJavaLexer extends GeneratedJavaLexer {
```

Comment:
it contains test methods which are used in this class, so I add them to idea TestMethodWithoutAssertion violations test method list